src/com/sun/jna/Native.java

```diff
@@ -952,6 +952,9 @@ private static void loadNativeDispatchLibrary() {
 
         String libName = System.getProperty("jna.boot.library.name", "jnidispatch");
         String bootPath = System.getProperty("jna.boot.library.path");
+        if (bootPath == null) {
+            bootPath = "/usr/lib/jni" + File.pathSeparator + "/usr/lib/" + Platform.getMultiArchPath() + "/jni";
+        }
```

Comment:
I read it, that you need this for debian packaging. If so, please keep the patch in debians packaging. It is a conscious decision not to rely on the system library. There were to many version mismatches, which cause issues. Debian can control both sides and it is fine to change this, for this distribution, I don't agree.

Reply:
OK I understand, I'll remove that.